requirements/requirements.common.txt

```diff
@@ -8,84 +8,84 @@ aiodns==2.0.0             # via pulpcore
+cffi==1.14.1              # via cryptography, pycares
 chardet==3.0.4            # via aiohttp, requests
-click==7.1.2              # via dynaconf, rq
+click==7.1.2              # via rq
 coreapi==2.3.3            # via drf-yasg
 coreschema==0.0.4         # via coreapi, drf-yasg
-cryptography==2.9.2       # via ansible
+cryptography==3.0         # via ansible
 defusedxml==0.6.0         # via odfpy
-diff-match-patch==20181111  # via django-import-export
+diff-match-patch==20200713  # via django-import-export
 django-filter==2.2.0      # via pulpcore
-django-import-export==2.0.2  # via pulpcore
+django-import-export==2.2.0  # via pulpcore
 django-prometheus==2.0.0  # via galaxy-ng (setup.py)
-django==2.2.13            # via django-filter, django-import-export, drf-nested-routers, drf-yasg, galaxy-ng (setup.py), pulpcore
+django==2.2.14            # via django-filter, django-import-export, drf-nested-routers, drf-spectacular, drf-yasg, galaxy-ng (setup.py), pulpcore
 djangorestframework-queryfields==1.0.0  # via pulpcore
-djangorestframework==3.10.3  # via drf-nested-routers, drf-yasg, pulpcore
+djangorestframework==3.10.3  # via drf-nested-routers, drf-spectacular, drf-yasg, pulpcore
 drf-nested-routers==0.91  # via pulpcore
+drf-spectacular==0.9.12   # via galaxy-ng (setup.py)
 drf-yasg==1.17.1          # via pulpcore
-dynaconf==2.2.3           # via pulpcore
+dynaconf==3.0.0           # via pulpcore
 et-xmlfile==1.0.1         # via openpyxl
-flake8==3.8.2             # via galaxy-importer
-galaxy-importer==0.2.4    # via pulp-ansible
+flake8==3.8.3             # via galaxy-importer
+galaxy-importer==0.2.7    # via pulp-ansible
 gunicorn==20.0.4          # via pulpcore
 idna-ssl==1.1.0           # via aiohttp
-idna==2.9                 # via idna-ssl, requests, yarl
-importlib-metadata==1.6.0  # via flake8, markdown
-inflection==0.4.0         # via drf-yasg
+idna==2.10                # via idna-ssl, requests, yarl
+importlib-metadata==1.7.0  # via flake8, jsonschema, markdown
+inflection==0.5.0         # via drf-spectacular, drf-yasg
 itypes==1.2.0             # via coreapi
 jdcal==1.4.1              # via openpyxl
 jinja2==2.11.2            # via ansible, coreschema
+jsonschema==3.2.0         # via drf-spectacular
 markdown==3.2.2           # via galaxy-importer
 markuppy==1.14            # via tablib
 markupsafe==1.1.1         # via jinja2
 mccabe==0.6.1             # via flake8
 multidict==4.7.6          # via aiohttp, yarl
 odfpy==1.4.1              # via tablib
-openpyxl==3.0.3           # via tablib
+openpyxl==3.0.4           # via tablib
 packaging==20.4           # via bleach, drf-yasg, pulp-ansible
 prometheus-client==0.8.0  # via django-prometheus
 psycopg2==2.8.5           # via pulpcore
-pulp-ansible==0.2.0b13    # via galaxy-ng (setup.py)
-pulpcore==3.4.1           # via galaxy-ng (setup.py), pulp-ansible
+pulp-ansible==0.2.0b15    # via galaxy-ng (setup.py)
+pulpcore==3.5.0           # via galaxy-ng (setup.py), pulp-ansible
```

Comment:
For the feature's I've ended up using in synclist stuff, I need pulpcore 3.6.x / master.

Reply:
iow, should this be pulpcore 3.6.x, or is this change intended for an interim release that wants 3.5.x?

Reply:
At the moment pulpcore 3.6.x does not exist. You may switch to local working copy from pulpcore master if you need.